src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -294,7 +304,12 @@ namespace dd
                 "timeseries not yet implemented over tensorRT backend");
           }
 
-        _nclasses = findNClasses(this->_mlmodel._def, _bbox);
+        if (_nclasses == 0)
```

Comment:
FTR, this works for now for classification / object detection models. For forthcoming OCR and regression models, this will need to be adapted. So we may want to put this into a function, now or later.

Reply:
What exactly do we want to put in a function?

Reply:
@beniz ?